### ISSUE TITLE
[BUGS] Release DMG contains non-working app, and gradlew build fails

### ISSUE THREAD

**Issue by 1dolla** [optyfr/JRomManager#286]
The DMG from the releases page contains an .app with an icon with the broken app symbol on top of it. Trying to run it causes "You can’t open the application “JRomManager.app” because this application is not supported on this Mac." (EDIT: This doesn't apply to the previous version, v3.0.3, just the more recent v3.0.4 (though they're both really v3.0.0? odd.))

macOS Sonoma 14.4.1, Intel

Using the repository with `./gradlew build` first fails as a JRM server test fails to bind to port 8080, which is the default port for the qBittorent WebUI, and thus in use on my system.

Working around this, it then fails in `winDistTar` for a reason that is more opaque to me:

```
> Task :winDistTar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':winDistTar' (type 'Tar').
  - Gradle detected a problem with the following location: '/Users/daniel/Projects/JRomManager/build/commons-daemon-1.3.3-bin-windows'.
    
    Reason: Task ':winDistTar' uses this output of task ':extractDaemonWin' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':extractDaemonWin' as an input of ':winDistTar'.
      2. Declare an explicit dependency on ':extractDaemonWin' from ':winDistTar' using Task#dependsOn.
      3. Declare an explicit dependency on ':extractDaemonWin' from ':winDistTar' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.7/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.7/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 13s
62 actionable tasks: 8 executed, 54 up-to-date
```

Cheers :)

**Comment by optyfr** [comment]
For gradlew, try this :
./gradlew jpackage -PtargetPlatform=macos-x64 -PinstallerType=dmg -xtest

I don't understand why 3.0.3 is working but not 3.0.4, nothing changed in the way of building dmg package between the two versions, I don't have a mac unfortunately to test (packages are built using github actions), I will try something because I see that jdk 14 is still used for packaging it should be jdk 21 (but it's same for 3.0.3 and former)

**Comment by optyfr** [comment]
I think I know why 3.0.4 is not working anymore : https://github.com/optyfr/JRomManager/issues/282#issuecomment-2290787836

**Comment by optyfr** [comment]
Should be solved with commit f50fdd5
